Functions/InputOutput/xASL_io_SplitASL.m

```diff
@@ -95,6 +95,9 @@ function xASL_io_SplitASL(inPath, iM0, iDummy)
 	% Check if both cbf and deltam/control/label volumes exist
 	if sum(~cellfun(@isempty,regexpi(aslContext,'deltam|control|label'))) && ...
 	   sum(~cellfun(@isempty,regexpi(aslContext,'cbf')))
+		% In this case, ignore CBF
+		warning('ASLContext has both a CBF image and delteM/control/label scans. CBF is thus ignored and discarded.')
```

Comment:
delteM -> deltaM

I would change this warning to:

"ASLContext has both a CBF image and more rawer ASL images (e.g., deltaM, control, label). CBF is thus discarded."

And similar to above: are we sure that we sync our image volume changes with the ASLContext changes? They should probably always be done side-to-side to avoid making debugging difficult.

Reply:
Yes - changing the comment. 

About syncing - done in another function. And you are right, this should be done and we have a new issue for this in 1.12.0. In 1.11.0 in this commit, we are only fixing the JSON information, which was incorrect. In the follow-up issue, we will do proper ASLContext, StudyPar, ASL.json synchronization.